src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -181,7 +182,7 @@ export class Inquiries extends React.Component<Props, State> {
     const ids = this.state.selected_conversations.map(value => value.id)
 
     const props = {
-      user_id: sharify.data.USER_DATA.id,
+      user_id: this.props.artsy.currentUser.id,
```

Comment:
No longer using the `sharify` singleton to access this data, but instead use the injected `artsy.currentUser` prop.